tests/integration/components/sl-radio-group-test.js

```diff
@@ -0,0 +1,169 @@
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="two"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="three"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    assert.equal(
+        this.$( 'input[readonly]' ).length,
+        3,
+        'Rendered component has three readonly inputs'
+    );
+});
+
+test( "Inline true sets sl-radio children's inline property to true", function( assert ) {
+    this.render( hbs`
+        {{#sl-radio-group inline=true name="testName"}}
+            {{#sl-radio label="One" value="one"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="two"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="three"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    assert.equal(
+        this.$( '.sl-radio.radio' ).length,
+        0,
+        'Rendered component children buttons are not inline'
+    );
+
+    assert.equal(
+        this.$( '.sl-radio.radio-inline' ).length,
+        3,
+        'Rendered component children buttons are inline'
+    );
+});
+
+test( "Inline false sets sl-radio children's inline property to false", function( assert ) {
+    this.render( hbs`
+        {{#sl-radio-group inline=false name="testName"}}
+            {{#sl-radio label="One" value="one"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="two"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="three"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    assert.equal(
+        this.$( '.sl-radio.radio-inline' ).length,
+        0,
+        'Rendered component has zero inline radio buttons'
+    );
+
+    assert.equal(
+        this.$( '.sl-radio.radio' ).length,
+        3,
+        'Rendered component has three default (non-inline) radio buttons'
+    );
+});
+
+test( 'Value changes when sl-radio child selected', function( assert ) {
+    this.set( 'value', 'jeremy' );
+    this.render( hbs`
+        {{#sl-radio-group value=value name="testName"}}
+            {{#sl-radio label="One" value="eric"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="josh"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="michael"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    Ember.run( () => {
+        const $radioButton = this.$( 'input[value="eric"]' );
+        $radioButton.click();
+    });
+
+    assert.equal(
+        this.get( 'value' ),
+        'eric',
+        '"eric" value is selected'
+    );
+
+    Ember.run( () => {
+        const $radioButton = this.$( 'input[value="josh"]' );
+        $radioButton.click();
+    });
+
+    assert.equal(
+        this.get( 'value' ),
+        'josh',
+        '"josh" value is selected'
+    );
+});
+
+test( 'default Value gets selected by default', function( assert ) {
+    this.set( 'value', 'eric' );
```

Comment:
Recommend changing the value setting to, and checking in the assertion, to "josh".  The reason for this is because some possible ways the code being tested could fail is that only the first item is checked for pre-selection and then the other elements are not in the rest of the loop running, or that the every item is set to be pre-selected.  In the first scenario the first radio button would be pre-selected and in the latter then last radio button would be but it would taken some special kind of code :-) for the middle option to be incorrectly targeted for pre-selection.